stable/kommander/values.yaml

```diff
@@ -53,9 +53,12 @@ federate:
   clusteraddons: true
   clusterrolebindings: true
   customresourcedefinitions: true
+  kubeaddons: true
   roles: true
   rolebindings: true
 
+  # set to true if you wish to deploy on a non konvoy cluster
```

Comment:
My one concern is that now we need to know whether we're deploying on Konvoy or non-Konvoy and can't use this as same resource(s) to federate to all joined clusters. Is there a way round this? As in, perhaps a pre-install hook that only creates kubeaddons namespace if required?

Reply:
alright, wasnt sure how we wanted to handle that. I will test a bit with a pre-install hook and get back to you. The word is that helm should not delete the resource in this case, which is what i would expect, but let me verify.

Reply:
could add it as an init container for kommander?

Reply:
@s12chung yep, digging into that possibility now.